### PR TITLE
All Azure + GCP partials reviewed

### PR DESCRIPTION
**Pull request by loic294**
# Description

**Make sure the PR is against the `develop` branch (see [Contributing](https://github.com/nccgroup/ScoutSuite/blob/master/CONTRIBUTING.md)).**

Please include a summary of the change(s) and which issue(s) it addresses. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change

Select the relevant option(s):

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (optional)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ScoutSuite/frontend/src/components/ResourceName/index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-  renderData: data => data.name,
+  renderData: data => data ? data.name : '',
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/components/ResourceName/index.js:16]: We could return the property `name` with `lodash/get` , so it also returns an empty string, `data` exists but not `name`.

File: ScoutSuite/frontend/src/pages/Findings/index.js [hunk 2]
```diff
@@ -27,7 +54,8 @@ const Findings = () => {
         <Breadcrumb />
         <div className="findings">
           <div className="table-card no-items">
-            <CheckCircleOutlineOutlinedIcon /> <b>All good!</b> No findings for this service.
+            <CheckCircleOutlineOutlinedIcon /> <b>All good!</b> No findings for
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/pages/Findings/index.js:57]: didnt Xavier want us to remove the "All good" part?

**Review by SamuelMeilleur — CHANGES_REQUESTED** [review]

File: ScoutSuite/frontend/src/components/ResourceName/index.js (hunk 1 above)
Comment (loic294) [reply ScoutSuite/frontend/src/components/ResourceName/index.js]: Fixed

File: ScoutSuite/frontend/src/pages/Findings/index.js (hunk 2 above)
Comment (loic294) [reply ScoutSuite/frontend/src/pages/Findings/index.js:57]: It's in my ohter PR: https://github.com/nccgroup/ScoutSuite/pull/1286

**Review by SamuelMeilleur — APPROVED** [review]

**Pull request merged**